// Proto file representing references to EDB Port Post Processing Prop

syntax = "proto3";

package ansys.api.edb.v1;

import "edb_messages.proto";

message PortPostProcessingPropMessage {
  ValueMessage voltage_magnitude = 1;
  ValueMessage voltage_phase = 2;
  ValueMessage deembed_length = 3;
  ValueMessage renormalization_impedance = 4;
  google.protobuf.BoolValue do_deembed = 5;
  google.protobuf.BoolValue do_deembed_gap_length = 6;
  google.protobuf.BoolValue do_renormalize = 7;
}
